README.md

```diff
@@ -143,7 +143,7 @@ Required:
 
 * numpy >= 1.18
 * scipy
-* astropy >= 3.2.3
+* astropy >= 4.2
```

Comment:
It won't let me insert a comment in the correct line, but the `Known Issues and Planned Improvements` section should be updated in this PR. I think the first two items should just be removed?

Reply:
Deleted 🎉